docs/quickstart/publish.md

```diff
@@ -5,88 +5,105 @@ description: 'learn about pinning services and the lifecycle of content-addresse
 
-After logging in, click on **Upload files** button to open the upload tab:
+When any file is successfully uploaded to IPFS (including the image used in this guide), the CID is generated. The CID for your uploaded image is: 
 
-![web3.storage upload button](./images/web3-upload-button.png)
+```plaintext
+bafybeicn7i3soqdgr7dwnrwytgq4zxy7a5jpkizrvhm5mv6bgjd32wm3q4
+```
 
-It should look as follows:
+You can now share the CID and anyone should be able to fetch it.
 
-![web3.storage upload tab](./images/web3-upload-tab.png)
+### What happens when you pin a file
 
-## 3. Upload and Pin the file
+When you upload and pin a file to web3.storage, the data is first transformed into a content-addressable representation, and then provided to the IPFS network:
 
-Either drag the [image file](/images/welcome-to-IPFS.jpg) to the dashed rectangle or click on the dashed rectangle to select the [image](/images/welcome-to-IPFS.jpg) file.
+1. **Representing:** The file was transformed into a content-addressable representation known as a [Merkle DAG](../concepts/merkle-dag.md). For the purpose of this guide, the full details are not important. The basic ideas is that this data representation makes files and directories **content-addressable** via CIDs by chunking files into smaller chunks and calculating their hashes.
+2. **Providing:** The content-addressable representation of the file was persisted on one of web3.storage's IPFS nodes (servers running an IPFS node) and made publicly available to the IPFS network.
 
-Once the file has been successfully uploaded you should see the following:
+### Fetching the CID from multiple gateways
 
-![web3.storage file uploaded](./images/web3-file-uploaded.png)
+IPFS Gateways are like bridges that give access to the IPFS Network using HTTP.
 
-If you close the upload tab you should also be able to see a shortened Content Identifier ([CID](../concepts/content-addressing.md)) of the uploaded image, `bafyb...d32wm3q4`:
+![gateway diagram](./images/gateway.png)
 
-![file after upload](./images/web3-file-after-upload.png)
+- [https://ipfs.io/ipfs/bafybeicn7i3soqdgr7dwnrwytgq4zxy7a5jpkizrvhm5mv6bgjd32wm3q4](https://ipfs.io/ipfs/bafybeicn7i3soqdgr7dwnrwytgq4zxy7a5jpkizrvhm5mv6bgjd32wm3q4)
+- [https://cloudflare-ipfs.com/ipfs/bafybeicn7i3soqdgr7dwnrwytgq4zxy7a5jpkizrvhm5mv6bgjd32wm3q4](https://cloudflare-ipfs.com/ipfs/bafybeicn7i3soqdgr7dwnrwytgq4zxy7a5jpkizrvhm5mv6bgjd32wm3q4)
+- [https://gateway.pinata.cloud/ipfs/bafybeicn7i3soqdgr7dwnrwytgq4zxy7a5jpkizrvhm5mv6bgjd32wm3q4](https://gateway.pinata.cloud/ipfs/bafybeicn7i3soqdgr7dwnrwytgq4zxy7a5jpkizrvhm5mv6bgjd32wm3q4)
 
-Congratulations, you have successfully **pinned** a file to IPFS!
+## Prerequisites
 
-### What's a CID
+- A free [web3.storage](https://web3.storage/) account.
+- The [following image](./images/welcome-to-IPFS.jpg), downloaded and saved on your computer:
```

Comment:
Btw, using a relative link here for a non-rendered image doesn't work. Hence the prior use of an absolute URL.